.travis.yml

```diff
@@ -21,10 +21,12 @@ script:
   - (cd libctl-src && git checkout master && sh autogen.sh --prefix=$HOME/local --enable-shared && make CFLAGS="-W" CXXFLAGS="-W" && make install)
   - git clone https://github.com/stevengj/harminv
   - (cd harminv && git checkout c221b2bcbaaa761f683aa5e2c6fa7efbbecdca1f && sh autogen.sh --prefix=$HOME/local --enable-shared && make && make install)
+  # test meep build with mpi, without python, omitting test suite
   - (CPPFLAGS=-I${HOME}/local/include LDFLAGS=-L${HOME}/local/lib GEN_CTL_IO=${HOME}/local/bin/gen-ctl-io sh autogen.sh --verbose --enable-maintainer-mode --prefix=$HOME/local --with-libctl=$HOME/local/share/libctl --with-mpi)
+  # test meep build without mpi, without python, with test suite
```

Comment:
It doesn't seem worthwhile to run the tests without python and mpi, and then with python but without mpi, because the latter tests are a strict superset of the former.  Just do the latter.

Reply:
Done.
I had to remove `near2far` from the travis test suite because it takes too long and the travis build always times out.